examples_drafts/gno.mod

```diff
@@ -1 +0,0 @@
-// Draft
```

Comment:
should be renamed to `gnomod.toml` and include `ignore = true` (and likely a path, as we are validating gnomod.toml files based on the presence of a path).

Reply:
febdaa2

should i refactor all the existing deprecated gno.mod
especially in:
- examples_ignored/...
- gnovm/tests/...